### ISSUE TITLE
[Bug] Custom TokenValidationParameters.ValidAudiences is overwritten

### ISSUE THREAD

**Issue by TiagoBrenck** [AzureAD/microsoft-identity-web#52]
**Which Version of Microsoft Identity Web are you using ?**
preview

**Where is the issue?**
* Web App
    * [ ] Sign-in users
    * [ ] Sign-in users and call web APIs
* Web API
    * [x] Protected web APIs (Validating tokens)
    * [ ] Protected web APIs (Validating scopes)
    * [ ] Protected web APIs call downstream web APIs
* Token cache serialization
     * [ ] In Memory caches
     * [ ] Session caches
     * [ ] Distributed caches

Other? - please describe;

**Repro**
As a developer, I might want to have my valid audience to be `https://<tenantDomain>` instead of `api://<guid>`. 

For that I could use:
```csharp
            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
                .AddProtectedWebApi("AzureAd", Configuration, options =>
                {
                    Configuration.Bind("AzureAD", options);
                    options.TokenValidationParameters.ValidAudiences = new List<string> { "https://<myTenantDomain>" };
                });
```

**Expected behavior**
Have the options.TokenValidationParameters.ValidAudiences set with my custom option.

**Actual behavior**
The method `EnsureValidAudiencesContainsApiGuidIfGuidProvided()` is overwriting my audiences in its logic. 

**Possible Solution**
Have `EnsureValidAudiencesContainsApiGuidIfGuidProvided()` to extend my options and not overwrite it.

**Additional context/ Logs / Screenshots**
Add any other context about the problem here, such as logs and screenshots.

**Comment by plamber** [comment]
Hello,
I have exactly this requirement. I fixed it temporarily by adding a property "Audience" in the MicrosoftIdentityOptions and the code at the end of "EnsureValidAudiencesContainsApiGuidIfGuidProvided"

if (!String.IsNullOrWhiteSpace(msIdentityOptions.Audience))
       validAudiences.Add(msIdentityOptions.Audience);

This ensures that the current code for existing solutions is not breaking and that I have the opportunity to add a custom audience in my config.

br,
Patrick

**Comment by jennyf19** [comment]
included in [0.1.1-preview release](https://github.com/AzureAD/microsoft-identity-web/releases/tag/0.1.1-preview)
